pkg/routeagent_driver/handlers/kubeproxy/endpoint_handler.go

```diff
@@ -30,6 +30,7 @@ func (kp *SyncHandler) LocalEndpointCreated(endpoint *submV1.Endpoint) error {
 	kp.syncHandlerMutex.Lock()
 	defer kp.syncHandlerMutex.Unlock()
 	kp.localCableDriver = endpoint.Spec.Backend
+	kp.localEndpoint = endpoint
```

Comment:
We don't need to cache the `localCableDriver` now that the whole endpoint is cached.